lib/core/src/server/common/babel.ts

```diff
@@ -31,12 +31,19 @@ export const plugins = [
    */
   require.resolve('@babel/plugin-proposal-optional-chaining'),
   require.resolve('@babel/plugin-proposal-nullish-coalescing-operator'),
+  [
+    require.resolve('babel-plugin-polyfill-corejs3'),
+    {
+      method: "usage-global",
+      absoluteImports: require.resolve('core-js')
+    }
+  ]
```

Comment:
@nicolo-ribaudo Is there a way to give it the version of `core-js` or does it figure that out itself? Not all versions of `core-js` have the same imports :thinking: